packages/fx-core/templates/plugins/resource/function/bicep/input_param.template.bicep

```diff
@@ -0,0 +1,7 @@
+
+param function_serverfarmsName string = '${resourceBaseName}-function-serverfarms'
+param function_webappName string = '${resourceBaseName}-function-webapp'
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for function backend.')
```

Comment:
What's the rule to add constraints and descriptions for parameters?

Reply:
I will remove them to keep all parameters aligned.